test/functionalTest/cases/3311_duplicate_metadata_in_filter/duplicate_metadata_in_filter.test

```diff
@@ -1,179 +0,0 @@
-# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
This .test is not related with subscriptions, so it should not be deleted in this PR.

(Maybe is has been removed due to my comment https://github.com/telefonicaid/fiware-orion/pull/4025#issuecomment-995033495. Sorry I actually mean duplicate_metadata_in_filter_sub.test...)

Reply:
Added test case in https://github.com/telefonicaid/fiware-orion/pull/4025/commits/7922dec240c6daecc35fcb2b4abec7fc1de3f1f0

Reply:
test/functionalTest/cases/3311_duplicate_metadata_in_filter/duplicate_metadata_in_filter.test stills appearing as deleted in "Files changed" tab.

![imagen](https://user-images.githubusercontent.com/1534240/146767067-7e385f31-c790-40c0-9d1c-0eccd407a822.png)

Please check.

Reply:
I have checked.  By mistake I have added another file sorry for this. Now I have added test/functionalTest/cases/3311_duplicate_metadata_in_filter/duplicate_metadata_in_filter.test in my last commit https://github.com/telefonicaid/fiware-orion/pull/4025/commits/3c4540e36aa51cd8c73feea618af899cfe3d43b6

Please check "Files changed" tab.